bin/ovn-k8s-overlay

```diff
@@ -416,33 +421,35 @@ def main():
     parser_minion_init.add_argument('--minion-switch-subnet', required=True,
                                     help="The smaller subnet just for this "
                                     "master.")
+    parser_minion_init.add_argument('--service-ip-subnet', required=True,
+                                    help="Subnet for kubernetes services.")
     parser_minion_init.add_argument('--node-name',
                                     required=True,
                                     help="A unique node name.")
     parser_minion_init.set_defaults(func=minion_init)
 
     # Parser for sub-command 'gateway-init'.
-    parser_minion_init = subparsers.add_parser(
+    parser_gateway_init = subparsers.add_parser(
```

Comment:
thanks for spotting this bad reuse of a local variable.
Personally, I would do a separate PR for this, but I do not have a strong opinion on this regard.